python/ray/data/tests/test_dataset_formats.py

```diff
@@ -1,58 +1,26 @@
 import os
-import shutil
```

Comment:
There's a recently-added `test_read_tfrecords`. Could you merge master in and check that it's here? Want to make sure no tests go missing.